src/redux/navigation.js

```diff
@@ -0,0 +1,10 @@
+module.exports.navigationReducer = function (state, action) {
+    if(typeof state = 'undefined') {
+        state = '';
+    }
+    return state;
```

Comment:
This is covers the base cases that we need of:
* getting an undefined state
* getting a state that is not used by this particular reducer

However, we still need to handle the case in which we get a state that comes from the `setSearchTerm` action below (i.e. some object with a unique `type` and `searchTerm` specified as keys). See [`conference-details`](https://github.com/LLK/scratch-www/blob/develop/src/redux/conference-details.js#L10-L24) for an example – we don't need as many possible states as that has, but something like `SET_DETAILS` but for the search term would be good (and maybe a `KeyMirror` object to set it up as well).